Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM bitnami/spark:3.5.1
 SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
 
 # Switch to root user so we can install jq and yq for parsing JSON/YAML
+# hadolint ignore=DL3002
```

Comment:
GitHub superlinter gets mad about not having a non-root user specified in the Dockerfile.